### ISSUE TITLE
Is there any tutorial for the DotTool?

### ISSUE THREAD

**Issue by Chromowolf** [Buizz/EUD-Editor-3#47]
![image](https://user-images.githubusercontent.com/37732663/116510431-7ba91880-a8f7-11eb-8bed-de038a8d55fd.png)

**Comment by Buizz** [comment]
This is a simple basic usage in Korean.
https://cafe.naver.com/edac/94767

**Comment by Chromowolf** [comment]
Thx a lot! But this article disallows all non-cafe users to view.
![image](https://user-images.githubusercontent.com/37732663/116820947-26486200-abaa-11eb-8f10-711a23c5ccdf.png)

![image](https://user-images.githubusercontent.com/37732663/116820845-af12ce00-aba9-11eb-9f7e-1672d6afde0d.png)
You know, as a non-Korean, it's nearly impossible to pass the verification procedure:
![image](https://user-images.githubusercontent.com/37732663/116820873-d8cbf500-aba9-11eb-9fab-02baf9a6c326.png)
Armoha told me a tutorial but it was too hard to follow:
https://blog.naver.com/fullhyuk/221462331342
I finally gave up. :(

In short, it's near impossible for me to view the content of https://cafe.naver.com/edac/94767. Could you please provide some screen shots of this tutorial? 1 screen shot is enough, as long as there is some text content (I could use the text content to search in naver)

**Comment by Chromowolf** [comment]
Ah, I somehow manged to view the tutorial.
But I was stuck at the picture importing step
![image](https://user-images.githubusercontent.com/37732663/116823787-197f3a80-abb9-11eb-8e60-4bcecb9edf0f.png)
How can I import a image of right size?
I tried to import the following 32x32 pixel PNG file, but there is no response in EE3:
![32x32](https://user-images.githubusercontent.com/37732663/116823818-4fbcba00-abb9-11eb-9aaf-fd828afeda3c.png)

**Comment by Buizz** [comment]
![image](https://user-images.githubusercontent.com/19189816/116868624-0c5e5c00-ac4a-11eb-8a91-abdc1c0dc1f6.png)

Doesn't this window appear?
Or is it popped up but there is no response when you select an image?

**Comment by Chromowolf** [comment]
It pops up but no response after selecting an image.
![image](https://user-images.githubusercontent.com/37732663/116874479-fa7db880-ac4b-11eb-9952-0f98fe5e466c.png)

No response:

![image](https://user-images.githubusercontent.com/37732663/116874557-16815a00-ac4c-11eb-9035-480e1000ed3e.png)

**Comment by Buizz** [comment]
Close for issues cleanup.
If the problem is not solved yet, please post a new issue.
